public/js/index.js

```diff
@@ -35,27 +37,28 @@ function displayArticle(article) {
         (
             '<a class="text-white" href="./views/produit.html?id=' 
             + article._id 
-            + '"><figure><img class="picture rounded-3 w-100" src="' 
+            + '"><figure><img class="picture rounded-3 w-100" src="'
             + article.imageUrl 
-            + '" alt="' 
+            + '" alt="'
             + article.name 
             + '"><figcaption class="w-100"><h2>' 
             + article.name 
-            + '</h2><p class="text-white">' 
+            + '</h2><p class="text-white">'
             + article.description 
-            + '</p><p class="my-5 text-white">Prix: ' 
+            + '</p><p class="my-5 text-white">Prix: '
             + convertPrice(article) 
-            + '</p></figcaption><p>Decouvrez ce modèle</p></figure></a>'
+            + '</p></figcaption><p>Découvrez ce modèle</p></figure></a>'
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/JulienBrazzalotto/JulienBrazzalotto_5_240921/pullRequest?prid=8291925)
```suggestion
            + "</p></figcaption><p>Découvrez ce modèle</p></figure></a>"
```